From a0708b844a0946f7e1ea6c6996d93f89ead32c7f Mon Sep 17 00:00:00 2001
From: marcone <48169102+marcone@users.noreply.github.com>
Date: Tue, 28 Jan 2025 15:40:59 -0800
Subject: [PATCH] README: show rpi-6.12.y build status

Remove rpi-5.15.y build status since it doesn't appear to be built anymore, and add rpi-6.12.y build status.
---
 README.md | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

--- a/README.md
+++ b/README.md
@@ -17,10 +17,6 @@ Please read the Documentation/process/ch
 requirements for building and running the kernel, and information about
 the problems which may result by upgrading your kernel.
 
-Build status for rpi-5.15.y:
-[![Pi kernel build tests](https://github.com/raspberrypi/linux/actions/workflows/kernel-build.yml/badge.svg?branch=rpi-5.15.y)](https://github.com/raspberrypi/linux/actions/workflows/kernel-build.yml)
-[![dtoverlaycheck](https://github.com/raspberrypi/linux/actions/workflows/dtoverlaycheck.yml/badge.svg?branch=rpi-5.15.y)](https://github.com/raspberrypi/linux/actions/workflows/dtoverlaycheck.yml)
-
 Build status for rpi-6.1.y:
 [![Pi kernel build tests](https://github.com/raspberrypi/linux/actions/workflows/kernel-build.yml/badge.svg?branch=rpi-6.1.y)](https://github.com/raspberrypi/linux/actions/workflows/kernel-build.yml)
 [![dtoverlaycheck](https://github.com/raspberrypi/linux/actions/workflows/dtoverlaycheck.yml/badge.svg?branch=rpi-6.1.y)](https://github.com/raspberrypi/linux/actions/workflows/dtoverlaycheck.yml)
@@ -28,3 +24,7 @@ Build status for rpi-6.1.y:
 Build status for rpi-6.6.y:
 [![Pi kernel build tests](https://github.com/raspberrypi/linux/actions/workflows/kernel-build.yml/badge.svg?branch=rpi-6.6.y)](https://github.com/raspberrypi/linux/actions/workflows/kernel-build.yml)
 [![dtoverlaycheck](https://github.com/raspberrypi/linux/actions/workflows/dtoverlaycheck.yml/badge.svg?branch=rpi-6.6.y)](https://github.com/raspberrypi/linux/actions/workflows/dtoverlaycheck.yml)
+
+Build status for rpi-6.12.y:
+[![Pi kernel build tests](https://github.com/raspberrypi/linux/actions/workflows/kernel-build.yml/badge.svg?branch=rpi-6.12.y)](https://github.com/raspberrypi/linux/actions/workflows/kernel-build.yml)
+[![dtoverlaycheck](https://github.com/raspberrypi/linux/actions/workflows/dtoverlaycheck.yml/badge.svg?branch=rpi-6.12.y)](https://github.com/raspberrypi/linux/actions/workflows/dtoverlaycheck.yml)
